python/runtime/modelzoo/load.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
```

Comment:
Just put it into `runtime/model` as `modelzoo.py` to avoid too many new packages.